src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 // write your code here
+const population = document.querySelectorAll('span.population');
+const totalElement = document.querySelector('span.total-population');
+const averageElement = document.querySelector('span.average-population');
+const populationNums = [];
+
+for (const populationElement of population) {
+  populationNums.push(+populationElement.innerText.split(',').join(''));
```

Comment:
1 - use `Number` instead of `+`, it's more readable
2 - use map instead of loop